01_the_machine_learning_landscape.ipynb

```diff
@@ -1,12 +1,30 @@
 {
```

Comment:
<p>AGAIN - # change v1 on a code block -&gt; touched in v2</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/handson-ml/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>